### PR TITLE
[tts][Non-ACR] Add extra descriptions for TTS modes

### PR DESCRIPTION
**Pull request by syeonholic**
### Change Description ###

 - Add extra descriptions for TTS modes

### CODE REVIEW

**Review by stom-hwang — APPROVED** [review]
LGTM 👍

File: docs/application/native/guides/text-input/tts.md [hunk 1]
```diff
@@ -452,10 +452,15 @@ To obtain the current state, the supported voice list, and the current voice, fo
 <a name="mode"></a>
 ## Get and set the mode
 
-There are 3 different TTS modes available. The main difference is audio mixing with other sources. The default mode is `TTS_MODE_DEFAULT`, used for normal applications such as eBooks. If you set this mode and play your text, it can be interrupted when other sounds, such as ringtones or other TTS sounds, are played.
+There are 3 different TTS modes available. 
+
+- `TTS_MODE_DEFAULT`: This mode is used for normal applications such as eBooks. If you do not set TTS mode, it will be set as `TTS_MODE_DEFAULT`.
+- `TTS_MODE_SCREEN_READER`: This mode is used for reading texts on screen such as Accessibility menu. This is dependent on whether Accessibility menu is turned on or not. If the Accessibility is OFF, TTS of an application, which sets TTS mode as `TTS_MODE_SCREEN_READER`, will not be operated.
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:458]:
```suggestion
- `TTS_MODE_SCREEN_READER`: This mode is used for reading texts on screen such as Accessibility Menu. This is dependent on whether Accessibility Menu is turned on or not. If the Accessibility is off, TTS of an application, which sets TTS mode as `TTS_MODE_SCREEN_READER`, will not be operated.
```

File: docs/application/native/guides/text-input/tts.md [hunk 2]
```diff
@@ -452,10 +452,15 @@ To obtain the current state, the supported voice list, and the current voice, fo
...
+- `TTS_MODE_NOTIFICATION`: This mode is used for notification such as pop-up.
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:459]:
```suggestion
- `TTS_MODE_NOTIFICATION`: This mode is used for notifications such as pop-up. 
```

**Review by omar-srbd — COMMENTED** [review]
Hello @syeonholic, I have reviewed the PR and left some suggestions for the text.
Please check and incorporate them.
Thank you.

File: docs/application/native/guides/text-input/tts.md [hunk 3]
```diff
@@ -452,10 +452,15 @@ To obtain the current state, the supported voice list, and the current voice, fo
 <a name="mode"></a>
 ## Get and set the mode
 
-There are 3 different TTS modes available. The main difference is audio mixing with other sources. The default mode is `TTS_MODE_DEFAULT`, used for normal applications such as eBooks. If you set this mode and play your text, it can be interrupted when other sounds, such as ringtones or other TTS sounds, are played.
+There are 3 different TTS modes available.
```
[the remark this answers is not in the document]
Comment (syeonholic) [reply docs/application/native/guides/text-input/tts.md:455]: Thank you for your suggestion. I added your suggestion. Please check the latest commit.

File: docs/application/native/guides/text-input/tts.md [hunk 4]
```diff
@@ -452,10 +452,15 @@ To obtain the current state, the supported voice list, and the current voice, fo
 <a name="mode"></a>
 ## Get and set the mode
 
-There are 3 different TTS modes available. The main difference is audio mixing with other sources. The default mode is `TTS_MODE_DEFAULT`, used for normal applications such as eBooks. If you set this mode and play your text, it can be interrupted when other sounds, such as ringtones or other TTS sounds, are played.
+There are 3 different TTS modes available. 
+
+- `TTS_MODE_DEFAULT`: This mode is used for normal applications such as eBooks. If you do not set TTS mode, it will be set as `TTS_MODE_DEFAULT`.
```
[the remark this answers is not in the document]
Comment (syeonholic) [reply docs/application/native/guides/text-input/tts.md:457]: Done.

File: docs/application/native/guides/text-input/tts.md (hunk 2 above)
Comment (syeonholic) [reply docs/application/native/guides/text-input/tts.md:459]: Done

File: docs/application/native/guides/text-input/tts.md [hunk 5]
```diff
@@ -452,10 +452,15 @@ To obtain the current state, the supported voice list, and the current voice, fo
...
+- `TTS_MODE_SCREEN_READER`: This mode is used for reading texts on screen such as Accessibility menu. This is dependent on whether Accessibility menu is turned on or not. If the Accessibility is OFF, TTS of an application, which sets TTS mode as `TTS_MODE_SCREEN_READER`, will not be operated.
+- `TTS_MODE_NOTIFICATION`: This mode is used for notification such as pop-up. 
+
 
 > [!NOTE]
-> The `TTS_MODE_NOTIFICATION` and `TTS_MODE_SCREEN_READER` modes are mixed with other sound sources, but they are used only for platform-specific features. Do not use them for normal applications.
+> When the 1st TTS request is playing, if the 2nd TTS request is coming, playing the 1st TTS request will be stopped and the 2nd TTS request will be played. All modes are mixed with other sound sources.
```
[the remark this answers is not in the document]
Comment (syeonholic) [reply docs/application/native/guides/text-input/tts.md:463]: Thank you for your suggestion. It seems better for users to understand the guide. 👍

**Review by omar-srbd — COMMENTED** [review]
Hello, @syeonholic. Please check the suggestion from the review and incorporate it.
Thank you.

File: docs/application/native/guides/text-input/tts.md [hunk 6]
```diff
@@ -452,10 +452,15 @@ To obtain the current state, the supported voice list, and the current voice, fo
 <a name="mode"></a>
 ## Get and set the mode
 
-There are 3 different TTS modes available. The main difference is audio mixing with other sources. The default mode is `TTS_MODE_DEFAULT`, used for normal applications such as eBooks. If you set this mode and play your text, it can be interrupted when other sounds, such as ringtones or other TTS sounds, are played.
+There are 3 different TTS modes available. They are described below:
+
+- `TTS_MODE_DEFAULT`: This mode is used for applications such as eBooks. If you do not set TTS mode, it will be set as `TTS_MODE_DEFAULT`.
```
[the remark this answers is not in the document]
Comment (syeonholic) [reply docs/application/native/guides/text-input/tts.md]: Thank you for suggestion!! I removed the empty line. Please review the latest commit again.

**Review by omar-srbd — APPROVED** [review]
@syeonholic, thanks for committing all the changes from the reviews.

@asm-foysal, this PR looks good to be merged. Hence, I am approving it.

**Pull request merged**